draft-shafranovich-rfc4180-bis.md

```diff
@@ -124,18 +124,30 @@ For example:
    zzz,yyy,xxxCRLF
 
 6. Fields containing line breaks (CR, LF or CRLF), double quotes, or commas
-MUST be enclosed in double-quotes. For example:
+MUST be enclosed in double-quotes. The same applies for the first field of
+a record that starts with a hash. For example:
 
    "aaa","b CRLF<br/>
    bb","ccc"CRLF<br/>
    zzz,yyy,xxxCRLF
+   "#aaa",#bbb,cccCRLF
 
 7. If double-quotes are used to enclose fields, then a double-quote
 appearing inside a field MUST be escaped by preceding it with
 another double quote. For example:
 
    "aaa","b""bb","ccc"CRLF
 
+8. A hash sign MAY be used to mark lines that are meant to be commented lines.
```

Comment:
Wondering about the language of "line" vs "record" here

Reply:
A line is terminated by an end-of-line character. A record might be comprised of multiple lines. A comment is not a record nor has any fields – it simply ends with an end-of-line and thus is a line.